### PR TITLE
UIU-2988 - User - Open Loans: UI changes for supporting DCB circulation

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
Disable open loan action buttons and hide open loan record action menu items for DCB Lending library

## Purpose
UIU-2988 - https://issues.folio.org/browse/UIU-2988

## Approach
Virtual user and virtual items are considered to determine DCB library roles.
Virtual User or DCB user will have lastName hardcoded to "DcbSystem" and type would be 'dcb'
Virtual Item would have hard coded instance id and holdings record id

#### Screencast
borrowing-pickup - open loans page
borrowing-pickup-role-open-loans

lending role - open loans page
lending-role-open-loans

pickup role - open loans page
pick-role-open-loans

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 
 "@aashutoshrathi/word-wrap@^1.2.3", "@aashutoshrathi/word-wrap@^1.2.6", "word-wrap@npm:@aashutoshrathi/word-wrap@^1.2.6":
+  name word-wrap
```
Comment (Dmytro-Melnyshyn) [inline yarn.lock:6]: Is this necessary?

Comment (Terala-Priyanka) [reply yarn.lock:6]: yes, without which lint script is not working

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Review by ryandberger — APPROVED** [review]

**Review by zburke — COMMENTED** [review]
This function should have been named `isDcbItem` to be consistent with the existing `isDcbUser`. I realize it seems petty, but inconsistencies like this add tiny amounts of friction that gradually overwhelm. Can you fix this?

**Comment by Terala-Priyanka** [comment]
> This function should have been named `isDcbItem` to be consistent with the existing `isDcbUser`. I realize it seems petty, but inconsistencies like this add tiny amounts of friction that gradually overwhelm. Can you fix this?

@zburke  https://issues.folio.org/browse/UIU-3020 has been created.

**Pull request merged**